src/commands/apps/create.js

```diff
@@ -54,6 +54,12 @@ async function addAddons (heroku, app, addons) {
   }
 }
 
+async function addConfigVars (heroku, app, configVars) {
+  await cli.action('Setting config vars', heroku.patch(`/apps/${app.name}/config-vars`, {
```

Comment:
Do you think it would be good to do something like `if (Object.keys(configVars).length === 0) return` to avoid making the call if there aren't any config vars?

Reply:
Oh, yeah, probably. Thanks for catching that.